# Author: Dragan Marinkovic <dmarinkovi@sierrawireless.com>
# Date:   Thu Now 26 14:20:19 2020 +0800
# audio: Startup sequence cleanup
#
# Fix the following complaints when audio kernel modules are loaded:
#
# msm-pcm-routing soc:qcom,msm-pcm-routing: ASoC: mux AUDIO_REF_EC_UL28 MUX has no paths
# msm-pcm-routing soc:qcom,msm-pcm-routing: ASoC: mux AUDIO_REF_EC_UL29 MUX has no paths
#
diff --git a/asoc/msm-pcm-routing-v2.c b/asoc/msm-pcm-routing-v2.c
index bf2bf1b4..88a61bb8 100644
--- a/asoc/msm-pcm-routing-v2.c
+++ b/asoc/msm-pcm-routing-v2.c
@@ -19949,10 +19949,6 @@ static const struct snd_soc_dapm_widget msm_qdsp6_widgets[] = {
 		&ext_ec_ref_mux_ul18),
 	SND_SOC_DAPM_MUX("AUDIO_REF_EC_UL19 MUX", SND_SOC_NOPM, 0, 0,
 		&ext_ec_ref_mux_ul19),
-	SND_SOC_DAPM_MUX("AUDIO_REF_EC_UL28 MUX", SND_SOC_NOPM, 0, 0,
-		&ext_ec_ref_mux_ul28),
-	SND_SOC_DAPM_MUX("AUDIO_REF_EC_UL29 MUX", SND_SOC_NOPM, 0, 0,
-		&ext_ec_ref_mux_ul29),
 };

 static const struct snd_soc_dapm_route intercon[] = {
